scriptworker/task.py

```diff
@@ -312,14 +348,46 @@ def get_repo_scope(task, name):
         return repo_scopes[0]
 
 
-# is_try {{{1
-def _is_try_url(url):
+def extract_github_repo_owner_and_name(repo_url):
+    """Given an URL, return the repo name and who owns it.
+
+    Args:
+        repo_url (str): The URL to the GitHub repository
+
+    Raises:
+        ValueError: on repo_url that aren't from github
+
+    Returns:
+        str, str: the owner of the repository, the repository name
+
+    """
+    if not repo_url.startswith('https://github.com/'):
+        raise ValueError('{} is not a supported GitHub URL!'.format(repo_url))
+
+    parts = _get_parts_of_url_path(repo_url)
```

Comment:
If you change it to 
```suggestion
    repo_owner, repo_name = _get_parts_of_url_path(repo_url)
```
then it will automatically error if there are not exactly two parts, which is correct.

Reply:
That doesn't work if the URL is something like: https://github.com/JohanLorenzo/android-components/raw/0e8222165d3ec11e932d5f900e8851dc98f62e98/.taskcluster.yml (tested at https://github.com/mozilla-releng/scriptworker/pull/307/files#diff-6e00aba980aa1d91133b0b8bfe689d88R198), which is still valid, IMO. Maybe it shouldn't. In which case, I'd prefer to handle this in a followup